### ISSUE TITLE
insert license: support for using tabs instead of spaces

### ISSUE THREAD

**Issue by benthevining** [Lucas-C/pre-commit-hooks#24]
In the insert-license hook, it would be great if these was an option for using tabs to indent instead of spaces.

Thanks!

**Comment by Lucas-C** [comment]
Hi @benthevining!
Could you please be more specific?
The [insert-license hook](https://github.com/Lucas-C/pre-commit-hooks#insert-license) makes no distinction between spaces & tabs.

**Comment by benthevining** [comment]
If I want the comment prefix in C++ multiline comments to be blank, ie:
```yaml
- repo: https://github.com/Lucas-C/pre-commit-hooks
    rev: v1.1.10
    hooks:
      - id: insert-license
        name: "Insert license header in C++ source files"
        args: [--license-filepath=util/lemons_header.txt,
               '--comment-style=/*|  | */',
               --detect-license-in-X-top-lines=16]
```

then the license header seems to be indented using space characters. This isn't a super pressing issue, though, I can live with using `*` as the comment prefix for now.

**Comment by benthevining** [comment]
Thank you!!!

**Comment by Lucas-C** [comment]
Released in [v1.1.11](https://github.com/Lucas-C/pre-commit-hooks/releases/tag/v1.1.11)
